include/openssl/crypto.h

```diff
@@ -175,6 +175,11 @@ OPENSSL_EXPORT unsigned long OpenSSL_version_num(void);
 
 OPENSSL_EXPORT unsigned long awslc_api_version_num(void);
 
+// |CRYPTO_mem_ctrl| intentionally does nothing.
+// AWS-LC does not support the related
+// memory debugging with |OPENSSL_NO_CRYPTO_MDEBUG| defined.
```

Comment:
Could be clearer that this is defined by default in AWS-LC. The original wording leaves some room for interpretation that this is adjustable by the user.
```suggestion
// AWS-LC defines `OPENSSL_NO_CRYPTO_MDEBUG` by default. These are related to
// memory debugging functionalities provided by OpenSSL, but are not supported in AWS-LC..
```

Reply:
We should also update the documentation we have listing no-op functions in our repo.  

https://github.com/aws/aws-lc/blob/3d3d3a3cf4f714e7c3d0189c3619ee3e2bd2ccbe/docs/porting/functionality-differences.md#libcrypto-no-ops

Reply:
@kingstjo - in this PR, would you also add this function to the listing of no-op functions in our documentation linked above? Thanks!

Reply:
Thank you both!
I have updated functionality-differences.md to reflect this no-op in addition to previous no-op `X509_TRUST_cleanup`